yesense_pub:
    ros__parameters:
        serial_port: "/dev/wheeltec_IMU"
        baud_rate: 460800
        frame_id: "gyro_link"
        driver_type: "linux_serial"
        imu_topic_ros: "imu/data_raw"
        imu_topic: "imu_data"


